pkg/apiserver/patch/patch.go

```diff
@@ -0,0 +1,454 @@
+/*
```

Comment:
Good point. I'm thinking I should rename it from Kube Merge Patch to something more generic as well, since the way it's implemented and its interface doesn't actually have anything to do with Kubernetes. Maybe Strategic Patch or something like that.